runtime/Jakt/String.h

```diff
@@ -39,7 +39,7 @@ class StringStorage : public RefCounted<StringStorage> {
 
     char const& operator[](size_t i) const
     {
-        VERIFY(i < m_length);
+        VERIFY(i <= m_length);
```

Comment:
This change isn't correct, it was right before. The character at `m_length` is the null byte, you should never be accessing that. If you are hitting an assertion, the problem is in your code or in the compiler, not here.